data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+	name "<planet> Mystery Man"
+	minor
+	description "Travel to <destination>, and meet up with Hroar in the spaceport."
+	to offer
+		has	"Nifel"
+	source "Valhalla"
+	destination "Muspel"
+	on offer
+		conversation
+			`The Transport Union employee is waiting for you near the entrance. He's wearing a wide-brimmed hat, a bulky scarf, a long coat with an up-turned collar, and sunglasses. You worry his over-the-top efforts to look nondescript will only draw more attention.`
+			`	"Can't speak long," he says. "It's too dangerous. I can't afford to be seen with you."`
+			`	You ask him what this is all about.`
+			`	"I don't agree with what the Union or The Deep are doing. I was raised in a small town where we still respected our past, and I couldn't just sit around and do nothing. So listen: with the Union against you, there's no transport company that will pick up your contract, with one exception. There's an old renegade mineral baron on <planet>. He hates the Union, and may be willing to do what you ask."`
+			choice
+				`	"How do I find him?"`
+			`	The Union employee begins moving away from you. "Name's Hroar. You can usually find him in the Red Rock Saloon, inside the Muspel Spaceport."`
+			`	The disgruntled Union employee then says, quite loudly, "No sir! I do not know where the bracelet shop is, please stop asking me!" And hurries away from you.`
+				accept
+	on complete
+		dialog
+			`You've arrived at the rusty-hued world that is Muspel. You should head into the spaceport to see if you can find Hroar.`
+
+
+
+mission "Home for Skadenga 9"
+	name "Home for Skadenga"
+	minor
+	invisible
+	to offer
+		has "Home for Skadenga 8: done"
+	source "Muspel"
+	destination
+		government "Republic" "Neutral" "Syndicate" "Free Worlds" "Deep"
+	on offer
+		event "hroar message" 13
+		conversation
+			`The floor of the spaceport is covered in the dirty red dust of hundreds of shuffling feet. Prospectors, miners, drillers, and all their attendant associates are intermixed here, gossiping about the latest finds, accidents, and hauls. Every person, either saturated in red earth or immaculately clean, creates a visual aid to boundaries of class, paycheck, and leadership.`
+			scene "scene/redrocks"
+			`	The Red Rock Saloon is located deep in the trade-craft section of the spaceport, far enough from the tourist areas to discourage anyone but the locals and those with business to enter.`
+			choice
+				`	(Enter the Red Rock Saloon.)`
+			`	The doors swing open, revealing a dimly lit room filled with rough unsavory people. Piano music is playing on a cheap speaker, and the eyes of the patrons slide to get a good look at you without turning their heads. There is an expectant pause. You get the feeling they are waiting for you to either announce something or leave.`
+			choice
+				`	"I'm looking for Hroar."`
+			`	For a moment nothing happens, and then a gruff voice spirals out from a dark corner. "Back here."`
+			scene "scene/hroar"
```

Comment:
I'm probably the most iffy on this picture. Nothing else I can think of in ES is hand-drawn. Again, I'd be curious to see what other think. Maybe it's just me.

Reply:
I will be replacing most, if not all, of the pictures. @roadrunner56 has offered to help in this.

Reply:
All images have been replaced with photographs.